contracts/validator-manager/interfaces/INativeTokenStakingManager.sol

```diff
@@ -24,6 +24,10 @@ interface INativeTokenStakingManager is IPoSValidatorManager {
         uint64 minStakeDuration
     ) external payable returns (bytes32 validationID);
 
+    /**
+     * @notice Begins the delegator registration process. Locks the provided native asset in the contract as the stake.
+     * @param validationID The ID of the validator to stake to.
```

Comment:
Not for this PR, but I notice we're not very consistent with how we refer to the entity represented by `validationID`. I like "validation period" since "validator" is easily confused with "nodeID" IMO, but don't feel strongly either way.